src/utils/validate-name.ts

```diff
@@ -10,7 +11,7 @@ type ValidateNpmNameResult =
     };
 
 export function validateNpmName(name: string): ValidateNpmNameResult {
-  const nameValidation = validateProjectName(name);
+  const nameValidation = validateProjectName(basename(resolve(name)));
```

Comment:
So then let's remove `basename(resolve(name))` from here?  https://github.com/scaffold-eth/create-eth/blob/main/src/utils/prompt-for-missing-options.ts#L29

Reply:
good point, thanks! Removed [81696ba](https://github.com/scaffold-eth/create-eth/pull/208/commits/81696baba8105d6e2a032e7ba8ebefcf97687432)